R/tests.R

```diff
@@ -269,6 +302,17 @@ testPlots <- function() {
         displayName = "PCA",
         plotType = "multiFeature",
         packages = "stats"
+      ),
+      plotMultiTest_sf = list(
```

Comment:
For consistency, please name your functions using camelCase: `plotMultiTestSf` and `plotMultiTestMf`

Reply:
This is implemented in the newest push - thanks for the suggestion